modules/misunderstood/src/hooks/after_incoming_middleware/00_misunderstood_flag.js

```diff
@@ -1,7 +1,27 @@
 const axios = require('axios')
 
 const flag = async () => {
-  if (event.type === 'text' && event.nlu.intent.name === 'none') {
+  const matchesQuickReply = async () => {
+    const { currentFlow } = event.state.context
+    if (currentFlow.startsWith('skills/choice')) {
```

Comment:
currentFlow could be undefined if it's the first message

Reply:
Fixed in 8bc5515